matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/widgets/WidgetPostAPIMediator.kt

```diff
@@ -65,7 +65,8 @@ interface WidgetPostAPIMediator {
     /**
      * Send an object response.
      *
-     * @param klass the class of the response
+     * @param T the generic type
```

Comment:
Not sure if  `* @param T the generic type` is needed

Reply:
yes, it has to be documented. The doc does not bring a lot TBH.